Formula/gnu-getopt.rb

```diff
@@ -1,31 +1,34 @@
+# This formula was previously based on the project hosted by the original
+# developer, which has since been merged into util-linux. The util-linux formula
+# could not be repurposed to provide getopt, though, since it only includes
+# tools that aren't already shipped with macOS. Thus, we now have two formulae
+# that offer independent subsets of util-linux
```

Comment:
This comment text should be removed from the formula.

You can add it in the commit message below the standard formatting